packages/atomic/src/components/atomic-smart-snippet/atomic-smart-snippet.stories.tsx

```diff
@@ -0,0 +1,44 @@
+import defaultStory from '../../../.storybook/default-story';
```

Comment:
👍 Great that you did not forget about Storybook !

Reply:
🤯 👍